tests/test_models/test_engine/test_db_storage.py

```diff
@@ -176,6 +176,22 @@ def test_key_format(self):
     @skipIf(storage_t != 'db', "not testing db storage")
     def test_storage_var_created(self):
         """ DBStorage object storage created """
-        from models.engine.db_storage import DBStorage
-        print(type(storage))
         self.assertEqual(type(storage), DBStorage)
+
+    def test_storage_get_method(self):
+        """tests FileStorage method to retrieve one object"""
+        # gets initial counts of all objects and all State objects in storage
+        count = storage.count()
+        count_state = storage.count(State)
+        # tests get method when id is not found
+        self.assertIs(storage.get(State, 89), None)
+        # creates new instance of State and saves to database
+        new = State()
+        new.save()
+        # tests if count increased for both all and State objects
```

Comment:
Why are we testing both the `count()` method and the `get()` method inside the same test? We should think about moving the count tests into a `test_storage_count_method()` function instead.

Reply:
Got these broken apart